index.html

```diff
@@ -13,7 +13,7 @@
             <section id="home">
                 <a href="#home" class="logo">
                     <img srcset="resources/logo@2x.png 2x" src="resources/logo.png" alt="Speedometer" />
-                    <div class="version">3.0</div>
+                    <div class="version next">Next</div>
```

Comment:
(optional) I would make it without a capital letters, all small letters, because the speedometer logo is also all small letters.
(maybe change the `alt` above to reflect that too)